pyrobuf/parse_proto.py

```diff
@@ -322,6 +363,10 @@ def _parse_message(self, current_message, tokens, messages, enums, imported_enum
             token.line + 1, self.lines[token.line])
 
         for token in tokens:
+            if self._handleComment(token, previous):
```

Comment:
To be more consistent with the subsequent code, I would prefer that we check `if token.token_type in ('OL_COMMENT', 'ML_COMMENT')` and call `_handle_comment` if true.

Reply:
That would not work than. Last comment must also be reset somewhere. That's why I did not call the function _parse_comment. If _handle_comment return with true, a comment has been parsed.